inlong-sort/sort-connectors/oracle-cdc/pom.xml

```diff
@@ -30,7 +30,7 @@
     <name>Apache InLong - Sort-connector-oracle-cdc</name>
 
     <properties>
-        <debezium.version>1.7.0.Final</debezium.version>
+        <debezium.version>1.6.4.Final</debezium.version>
```

Comment:
The version of oracle-cdc is upgraded but the version of debezium is lower?

Reply:
Flink cdc 2.3 depends on the 1.6.4 version of debezium, and the 1.7.0 version of debezium conflicts with it.